framework/include/SPIDevObj.hpp

```diff
@@ -49,12 +49,12 @@ class SPIDevObj : public DevObj
 {
 public:
 	enum SPI_FREQUENCY {
-		SPI_FREQUENCY_320KHZ = 320000UL,
-		SPI_FREQUENCY_1MHZ = 1000000UL,
-		SPI_FREQUENCY_5MHZ = 5000000UL,
-		SPI_FREQUENCY_10MHZ = 10000000UL,
-		SPI_FREQUENCY_15MHZ = 15000000UL,
-		SPI_FREQUENCY_20MHZ = 20000000UL,
+		SPI_FREQUENCY_320KHZ = 320000,
+		SPI_FREQUENCY_1MHZ = 1000000,
+		SPI_FREQUENCY_5MHZ = 5000000,
+		SPI_FREQUENCY_10MHZ = 10000000,
+		SPI_FREQUENCY_15MHZ = 15000000,
+		SPI_FREQUENCY_20MHZ = 20000000,
```

Comment:
Hm, I only see this one: https://github.com/nicolaerosia/DriverFramework/blob/07ce9bea892a5c595b57917403e7cebea273356a/framework/src/SPIDevObj.cpp#L456